src/docker-osx-dev

```diff
@@ -1142,12 +1151,22 @@ function handle_command {
   done
 
   case "$cmd" in
-    "$SYNC_COMMAND")
+    "$SYNC_COMMAND" | "$SYNC_ONLY_COMMAND" | "$WATCH_ONLY_COMMAND")
       configure_log_level "$log_level"
       configure_paths_to_sync "$docker_compose_file" "${paths_to_sync[@]}"
       configure_excludes "$ignore_file" "${excludes[@]}"
       configure_includes "$ignore_file" "${includes[@]}"
-      watch_and_sync
+
+      case "$cmd" in
```

Comment:
I changed it to two nested cases for clarity.  The first case will do the initialization, and the second will run the three scenarios from which we initialized.